package-lock.json

```
@@ -11,7 +11,7 @@
       "dependencies": {
         "classnames": "^2.3.2",
         "format-number": "^3.0.0",
-        "fundraising-frontend-content": "github:wmde/fundraising-frontend-content#production",
+        "fundraising-frontend-content": "github:wmde/fundraising-frontend-content#test",
```

should get switched back to production again right?
```suggestion
        "fundraising-frontend-content": "github:wmde/fundraising-frontend-content#production",
```